src/components/Calculator.js

```diff
@@ -1,76 +1,117 @@
 import React from 'react';
 import '../App.css';
+import calculate from '../logic/calculate';
```

Comment:
- [OPTIONAL] You don't need to include calculate in the file path. This is enough `../logic`